pkg/controller/clientutil/object.go

```diff
@@ -0,0 +1,14 @@
+package clientutil
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func ObjectsToClientObjects[T client.Object](objs []T) []client.Object {
```

Comment:
why not make these objs `client.Object` from the beginning? 
Does `kubernetes.Object` have more methods that we need to use before we get to this point?

Reply:
Ahh great question again, moved convo to https://github.com/aws/eks-anywhere/pull/3174#discussion_r955519684